index.js

```diff
@@ -12,8 +12,8 @@ function expressPlugin (fastify, options, next) {
   fastify.express.disable('x-powered-by')
 
   fastify
-    .addHook('onRequest', enhanceRequest)
-    .addHook('onRequest', runConnect)
+    .addHook('preHandler', enhanceRequest)
+    .addHook('preHandler', runConnect)
```

Comment:
Could you make the hook configurable via an option, defaulting to `onRequest`?

Reply:
Done. But not the best naming, I think..